src/ZenviaServiceProvider.php

```diff
@@ -20,6 +20,7 @@ public function boot()
                     $config['conta'],
                     $config['senha'],
                     isset($config['from'])    ? $config['from']    : null,
+                    $config['aggregateId'],
```

Comment:
Adicionaria a mesma condição `isset` aqui só pra garantir que não vai quebrar mesmo que não tenha essa opção nas configurações. Exemplo quem for atualizar a versão do pacote depois de fazer o merge desta PR.